src/guides/v2.4/security/two-factor-authentication.md

```diff
@@ -18,7 +18,6 @@ Two-Factor Authentication gives you the ability to:
 Magento has new workflows for _Admin_ users, including:
 
 -  The ability to configure the 2FA provider globally or individually.
```

Comment:
In order to remove concern regarding the individual selection of providers, I suggest making this correction:

```suggestion
- Initially configure the global 2FA providers.
```